deployment/terraform/utils.go

```diff
@@ -352,20 +355,116 @@ func getServerURL(output *Output, deploymentConfig *deployment.Config) string {
 	return deploymentConfig.ServerScheme + "://" + url
 }
 
-// GetAWSConfig returns the AWS config, using the profile configured in the
-// deployer if present, and defaulting to the default credential chain otherwise
-func (t *Terraform) GetAWSConfig() (aws.Config, error) {
+// Durations for the expiry of the AWS Role credentials and for its refresh
+// interval, which, out of an abundance of caution, is a quarter of the expiry
+// duration of the role.
+const (
+	// One hour is the maximum allowed by role chaining
+	AWSRoleTokenDuration        = 1 * time.Hour
```

Comment:
nit: 1 multiplier is redundant.

Reply:
Yeah, but `0 + time.Hour` was weirder.

Reply:
`0 + time.Hour` is much clearer! We should use that everywhere!